### PR TITLE
NOC-368-Fix for saving career profiles

### PR DESCRIPTION
**Pull request by sunanditasahu**

### CODE REVIEW

File: src/WorkBC.Data/Migrations/20240510201850_UpdateIndustryProfilesWithNoc2021.cs [hunk 1]
```diff
@@ -153,12 +153,8 @@ protected override void Down(MigrationBuilder migrationBuilder)
                 int industryProfileId = x.indProfile.IndustryProfileID;
 
                 //Restore the values of EDM_IndustryProfile_IndustryProfileId column in dbo.SavedIndustryProfiles table.
-                migrationBuilder.UpdateData(
-                table: "SavedIndustryProfiles",
-                keyColumn: "IndustryId",
-                keyValue: industryId,
-                column: "EDM_IndustryProfile_IndustryProfileId",
-                value: industryProfileId);
+                migrationBuilder.Sql("Update SavedIndustryProfiles SET EDM_IndustryProfile_IndustryProfileId =" +
```
Comment (infojunkie) [inline src/WorkBC.Data/Migrations/20240510201850_UpdateIndustryProfilesWithNoc2021.cs:156]: is this change intentional?

Comment (sunanditasahu) [reply src/WorkBC.Data/Migrations/20240510201850_UpdateIndustryProfilesWithNoc2021.cs:156]: yes

File: src/WorkBC.Data/Migrations/JobBoardContextModelSnapshot.cs [hunk 2]
```diff
@@ -1414,8 +1414,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("nvarchar(450)");
 
                     b.Property<int>("NocCodeId2021")
-                        .HasColumnType("int")
-                        .HasColumnName("NocCodeId2021");
+                        .HasColumnType("int");
```
[the remark this answers is not in the document]
Comment (sunanditasahu) [reply src/WorkBC.Data/Migrations/JobBoardContextModelSnapshot.cs:1417]: yes, to make it uniform with the other fields.
